README.md

```diff
@@ -9,14 +9,15 @@ This is the codebase for the Brainy Kids backend and administration panel. This
 1.  Make sure you have Node.js installed (Make sure `node --version` prints out a version number)
 2.  Make sure you have the latest version of MongoDB set up either locally or point it to a remote server (see "Getting Started")
 3.  If you want to install this for actual use, make sure you install it on a server that can be accessed by any computer that wants to use it. This usually means the server should have a static IP address and correct port forwarding/firewall configuration.
+4. You must have npm installed. Run `npm --version` in your terminal and expect to see a version number printed out
 
 ## Getting started
 
 1.  Clone this repository (`git clone git@github.com:jvt/Brainy-Kidz.git`)
 2.  Open the repository (`cd Brainy-Kidz`)
 3.  Install the dependencies (`npm install`)
 4.  Duplicate `.env.example` and rename it to `.env`
-5.  Open the `.env` file and add a JWT_SECRET
+5.  Open the `.env` file and add a JWT_SECRET. It can be any string.
```

Comment:
Add a second sentence here something like, `It should be a random series of letters / numbers. This string is used to encrypt all the tokens for access to the site`